### ISSUE TITLE
Can't install/build using pip

### ISSUE THREAD

**Issue by stevenleeg** [pistolero/python-scss#20]
Hey! For some reason I'm not able to build sass under debian sqeeze:

```
$ pip install sass

... compiler warnings for a while ...

cc1plus: warning: command line option "-Wstrict-prototypes" is valid for Ada/C/ObjC but not for C++

gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I. -Ilibsass -I/usr/include/python2.6 -c sass.c -o build/temp.linux-x86_64-2.6/sass.o

gcc: sass.c: No such file or directory

gcc: no input files

error: command 'gcc' failed with exit status 1
```

For reference, I'm running Python 2.6.6. Let me know what info I can provide to help get to the bottom of this!

---

I can also replicate this issue on OS X (10.9.2, python 2.7.6):

```
Downloading/unpacking sass
  Running setup.py egg_info for package sass
    No Cython installed. Building from pregenerated C source.

    warning: no files found matching '*.c'
Installing collected packages: sass
  Running setup.py install for sass
    No Cython installed. Building from pregenerated C source.
    building 'sass' extension
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/ast.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/ast.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/base64vlq.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/base64vlq.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/bind.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/bind.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/constants.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/constants.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/context.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/context.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/contextualize.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/contextualize.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/copy_c_str.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/copy_c_str.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/error_handling.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/error_handling.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/eval.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/eval.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/expand.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/expand.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/extend.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/extend.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/file.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/file.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/functions.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/functions.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/inspect.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/inspect.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/output_compressed.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/output_compressed.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/output_nested.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/output_nested.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/parser.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/parser.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/prelexer.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/prelexer.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sassc++.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sassc++.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sass.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sass.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sass_interface.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sass_interface.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/source_map.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/source_map.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/to_c.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/to_c.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/to_string.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/to_string.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/units.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/units.o
    clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c sass.c -o build/temp.macosx-10.9-x86_64-2.7/sass.o
    clang: error: no such file or directory: 'sass.c'
    clang: error: no input files
    error: command 'clang' failed with exit status 1
    Complete output from command /Users/steve/Desktop/tes/bin/python -c "import setuptools;__file__='/Users/steve/Desktop/tes/build/sass/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/ln/m3fwtf0s3z76k8x0m7d804lh0000gn/T/pip-G2jpxH-record/install-record.txt --single-version-externally-managed --install-headers /Users/steve/Desktop/tes/bin/../include/site/python2.7:
    No Cython installed. Building from pregenerated C source.

running install

running build

running build_ext

building 'sass' extension

creating build

creating build/temp.macosx-10.9-x86_64-2.7

creating build/temp.macosx-10.9-x86_64-2.7/libsass

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/ast.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/ast.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/base64vlq.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/base64vlq.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/bind.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/bind.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/constants.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/constants.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/context.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/context.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/contextualize.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/contextualize.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/copy_c_str.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/copy_c_str.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/error_handling.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/error_handling.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/eval.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/eval.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/expand.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/expand.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/extend.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/extend.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/file.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/file.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/functions.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/functions.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/inspect.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/inspect.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/output_compressed.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/output_compressed.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/output_nested.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/output_nested.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/parser.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/parser.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/prelexer.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/prelexer.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sassc++.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sassc++.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sass.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sass.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/sass_interface.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/sass_interface.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/source_map.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/source_map.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/to_c.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/to_c.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/to_string.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/to_string.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c libsass/units.cpp -o build/temp.macosx-10.9-x86_64-2.7/libsass/units.o

clang -fno-strict-aliasing -fno-common -dynamic -I/usr/local/include -I/usr/local/opt/sqlite/include -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I. -Ilibsass -I/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c sass.c -o build/temp.macosx-10.9-x86_64-2.7/sass.o

clang: error: no such file or directory: 'sass.c'

clang: error: no input files

error: command 'clang' failed with exit status 1

----------------------------------------
Cleaning up...
Command /Users/steve/Desktop/tes/bin/python -c "import setuptools;__file__='/Users/steve/Desktop/tes/build/sass/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/ln/m3fwtf0s3z76k8x0m7d804lh0000gn/T/pip-G2jpxH-record/install-record.txt --single-version-externally-managed --install-headers /Users/steve/Desktop/tes/bin/../include/site/python2.7 failed with error code 1 in /Users/steve/Desktop/tes/build/sass
Storing complete log in /Users/steve/.pip/pip.log
```

**Comment by andyparsons** [comment]
I'm experiencing the same issue. Any workaround?

**Comment by stevenleeg** [comment]
I believe I fixed this by installing Cython (`pip install cython`). Let me know if that helps!

**Comment by RomaDotDev** [comment]
I worked for me, thanks.

**Comment by pistolero** [comment]
Fixed in 2.2. Now `pip install sass` should work without `cython`.

**Comment by timtan** [comment]
Hello,  I meet almost the same issue, below are part of my error message. 

cc -fno-strict-aliasing -fno-common -dynamic -arch x86_64 -arch i386 -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -arch x86_64 -arch i386 -pipe -I. -Ilibsass -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c sass.c -o build/temp.macosx-10.9-intel-2.7/sass.o

clang: error: no such file or directory: 'sass.c'

clang: error: no input files

error: command 'cc' failed with exit status 1

---

I use Mac OS 10.9.3, 

here is my compiler information

 g++ -v
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 5.1 (clang-503.0.40) (based on LLVM 3.4svn)
Target: x86_64-apple-darwin13.2.0
Thread model: posix
